services/paymaster/src/rpc.rs

```diff
@@ -21,6 +21,15 @@ pub enum ConfirmationResult {
     },
 }
 
+impl ConfirmationResult {
```

Comment:
Would it be more idiomatic to make this `Display` or `ToString`?

Reply:
idk if that's the right use case for those traits since some info is being lost... this representation is just for the label string